src/main/kotlin/Calculator.kt

```diff
@@ -1,3 +1,61 @@
-class Calculator(val A: Float, val B: Float) {
-    fun Plus(): Float{return A + B}
-}
\ No newline at end of file
+class Calculator {
+    fun plus(leftOperand: Double, rightOperand:Double) =  leftOperand + rightOperand
+    fun minus(leftOperand: Double, rightOperand: Double) = leftOperand - rightOperand
+    fun multiply(leftOperand: Double, rightOperand: Double) = leftOperand * rightOperand
+    fun divide(leftOperand: Double, rightOperand: Double): Double?{
+        try {
+            if (rightOperand == 0.0)
+                throw Exception("Division by zero!\n")
+            else
+                return leftOperand/rightOperand
+        }
+        catch (e: Exception){
+            print(e.message)
+            return null
+        }

```

Comment:
It's different task 
Must be in different pull request
And that must be in own class
I made task for this